src/Foundation.Find.Cms/Facets/Config/FacetConfigFactory.cs

```diff
@@ -44,13 +46,15 @@ public virtual FacetDefinition GetFacetDefinition(FacetFilterConfigurationItem f
                     {
                         FieldName = facetConfiguration.FieldName,
                         DisplayName = facetConfiguration.GetDisplayName(),
+                        IsCommerceEnabled = true
```

Comment:
Should be false

Reply:
I updated FacetFilterConfigurationItem model and you need to update Search Settings as image below to enable content facets for Categories and Content Type:
![image](https://user-images.githubusercontent.com/56108286/76720244-d670c680-676e-11ea-8b8b-4c6ff6abbbec.png)